### DISCUSSION TITLE
Any progress on per-directory translation?

### DISCUSSION THREAD

**Discussion by mendax1234** [3ru/gpt-translate#13] [Ideas]
l think it is hard to use GitHub cli to read all the file's name under one directory, so l am curious about whether gpt cli can translate the per directory file into another directory.

**Comment by 3ru** [comment]
@mendax1234 
Supported in v1.1 🎉

This GitHub Actions is implemented in nodejs.
We can read directories using the read/write file module (fs).

**Comment by mendax1234** [comment]
That's great! @3ru

**Comment by 3ru** [comment]
In https://github.com/3ru/gpt-translate/releases/tag/v1.1.6, you can now specify files more generically:tada:
